package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
   "devDependencies": {
     "brunch": "^2.0.1",
+    "bower": "^1.6.0",
```

Comment:
@nclsHart @martin-damien What do you think of this approach http://www.joezimjs.com/javascript/no-more-global-npm-packages/ ?